gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -133,19 +134,18 @@ private void handleSaveResult(EditObservationViewModel.SaveResult saveResult) {
   }
 
   private void addFieldViewModel(Field field, AbstractFieldViewModel fieldViewModel) {
-    fieldViewModel.init(field, viewModel.getResponse(field.getId()));
+    // TODO: Get initial response via parameters.
```

Comment:
I see `getSavedOrOriginalResponse()` is still in the ViewModel. I understood we'd get rid of that in favor of the ViewModel containing the "current" state of the UI. Is this still what we want to do?